src/sentry/api/endpoints/api_tokens.py

```diff
@@ -46,7 +46,11 @@ def get(self, request: Request) -> Response:
                 "application"
             )
         )
-        # TODO: when the delete endpoint no longer requires the full token value, update this to stop including token
```

Comment:
lol

Reply:
Yeah for blue/green deployments, I'll have to do this in increments; will be tedious haha

Reply:
Good thing is that we can just update a param and it'll stop! So not too much